src/plugins/formulas/__tests__/formulas.spec.js

```diff
@@ -1048,30 +1023,299 @@ describe('Formulas general', () => {
       expect(hot.getSourceDataAtRow(1)).toEqual([2009, 0, 2941, 4303, 354, 5814]);
       expect(hot.getSourceDataAtRow(2)).toEqual([2010, 5, 2905, 2867, '=SUM(A4,2,3)', '=$B1']);
       expect(hot.getSourceDataAtRow(3)).toEqual([2011, 4, 2517, 4822, 552, 6127]);
-      expect(hot.getSourceDataAtRow(4)).toEqual([2012, '=Sum(a2:a5)', '=SUM(B5,E3)', '=A2/B2', 12, '\'=SUM(E5)']);
+      expect(hot.getSourceDataAtRow(4)).toEqual([2012, '=Sum(a2:a5)', '=SUM(B5,E3)', '=A2/B2', 12, '=SUM(E5)']);
 
       hot.redo();
 
       expect(hot.getSourceDataAtRow(0)).toEqual(['=$B$2', 'Maserati', 'Mazda', 'Mercedes', 'Mini', '=A$1']);
       expect(hot.getSourceDataAtRow(1)).toEqual([2009, 0, 2941, 4303, 354, 5814]);
       expect(hot.getSourceDataAtRow(2)).toEqual([2011, 4, 2517, 4822, 552, 6127]);
-      expect(hot.getSourceDataAtRow(3)).toEqual([2012, '=SUM(A2:A4)', '=SUM(B4,#REF!)', '=A2/B2', 12, '\'=SUM(E5)']);
+      expect(hot.getSourceDataAtRow(3)).toEqual([2012, '=SUM(A2:A4)', '=SUM(B4,#REF!)', '=A2/B2', 12, '=SUM(E5)']);
 
       hot.redo();
 
       expect(hot.getSourceDataAtRow(0)).toEqual(['=$B$2', 'Maserati', 'Mini', '=A$1']);
       expect(hot.getSourceDataAtRow(1)).toEqual([2009, 0, 354, 5814]);
       expect(hot.getSourceDataAtRow(2)).toEqual([2011, 4, 552, 6127]);
-      expect(hot.getSourceDataAtRow(3)).toEqual([2012, '=SUM(A2:A4)', 12, '\'=SUM(E5)']);
+      expect(hot.getSourceDataAtRow(3)).toEqual([2012, '=SUM(A2:A4)', 12, '=SUM(E5)']);
 
       hot.redo();
 
       expect(hot.getSourceDataAtRow(0)).toEqual([2011, 4, 552, 6127]);
-      expect(hot.getSourceDataAtRow(1)).toEqual([2012, '=SUM(A1:A2)', 12, '\'=SUM(E5)']);
+      expect(hot.getSourceDataAtRow(1)).toEqual([2012, '=SUM(A1:A2)', 12, '=SUM(E5)']);
+    });
+  });
+
+  describe('autofill', () => {
+    const fillHandleSelector = '.wtBorder.current.corner';
+
+    const autofill = (endRow, endCol) => {
+      spec().$container.find(fillHandleSelector).simulate('mousedown');
+
+      spec().$container
+        .find(`tbody tr:eq(${endRow}) td:eq(${endCol})`)
+        .simulate('mouseover')
+        .simulate('mouseup');
+    };
+
+    // Most of these tests will produce invalid values (out of bound addresses,
+    // #CYCLE! errors), but we only care about the formula offsets.
+    //
+    // https://docs.google.com/spreadsheets/d/1ERI3YEe7GYWUKdKGPU4C97yUh1fOM6HILZY03AB8wwk/edit?usp=sharing
```

Comment:
Sure in the simplest case it could be more readable, but imo it's just too much effort to fix the more complex ones as nobody should really look at the rendered output anyway, just the offsets described in code.

Reply:
Well, the commend put my focus on the matter. If it's ignored then let's omit it